docs/docs/installation/gitlab.md

```diff
@@ -42,21 +42,36 @@ Note that if your base branches are not protected, don't set the variables as `p
 
 ## Run a GitLab webhook server
 
-1. From the GitLab workspace or group, create an access token. Enable the "api" scope only.
+1. From the GitLab workspace or group, create an access token with "Reporter" role and "api" scope.
```

Comment:
should at least a 'developer'

https://docs.gitlab.com/ee/user/permissions.html


<img width="855" alt="image" src="https://github.com/user-attachments/assets/ef857c77-0b7a-47e3-868c-8b6b6836a777">

Reply:
I see. But wiki access is only possible with Pro features, right? I can detail that.

Reply:
ok, i guess you are right

Reply:
adjusted